server/src/main/java/io/druid/server/coordinator/rules/LoadRule.java

```diff
@@ -240,6 +240,13 @@ public void execute()
     return stats;
   }
 
+  protected void validateTieredReplicants(Map<String, Integer> tieredReplicants){
+    for(Map.Entry<String, Integer> entry: tieredReplicants.entrySet()){
+      if(entry.getValue() < 0)
+        throw new IllegalArgumentException("Replicant value cannot be less than 0");
```

Comment:
One other exception you can take a look at is IAE. We use it a bit more than IllegalArgumentException for the ability to more easily substitute string values. Not a big deal here.